lib/domain/app_state/search/search_interactor_state.dart

```diff
@@ -4,8 +4,14 @@ import 'package:fluffychat/domain/model/search/search_model.dart';
 
 class GetContactAndRecentChatSuccess extends Success {
   final List<SearchModel> search;
-
-  const GetContactAndRecentChatSuccess({required this.search});
+  final int contactsOffset;
+  final bool shouldLoadMoreContacts;
+
+  const GetContactAndRecentChatSuccess({
+    required this.search,
+    required this.contactsOffset,
+    required this.shouldLoadMoreContacts,
+  });
 
   @override
   List<Object?> get props => [search];
```

Comment:
```suggestion
  List<Object?> get props => [search, contactsOffset, shouldLoadMoreContacts];
```

IMO: Don't forget it

Reply:
done